misc/scripts/library-coverage/compare-files-comment-pr.py

```diff
@@ -41,15 +46,34 @@ def compare_files_str(file1, file2):
     return ret
 
 
-def comment_pr(folder1, folder2, output_file, repo, pr_number, run_id):
-    compare_folders(folder1, folder2, output_file)
+def write_diff_for_run(output_file, repo, run_id):
+    folder1 = "out_base"
+    folder2 = "out_merge"
+    try:
+        utils.subprocess_run(["gh", "run", "download", "--repo", repo, "--name",
```

Comment:
Minor (really very minor): You could do these in a short loop over `[folder1, folder2]`. Same for the cleanup below.

Reply:
I'm not entirely sure what you mean. The name of the artefacts are different, so a simple loop would not be enough. On the other hand, I've changed the cleanup to use a loop.